packages/smart-router/evm/index.test.ts

```diff
@@ -0,0 +1,43 @@
+import { expect, test } from 'vitest'
+import * as exports from './index'
+
+test('exports', () => {
+  expect(Object.keys(exports)).toMatchInlineSnapshot(`
+    [
+      "StableSwap",
+      "SmartRouter",
```

Comment:
Let's add snapshot test for SmartRouter and StableSwap as well

Reply:
done